ListableUI/Sources/Internal/DefaultSwipeView.swift

```diff
@@ -11,22 +11,64 @@ private let haptics = UIImpactFeedbackGenerator(style: .light)
 
 public final class DefaultSwipeActionsView: UIView, ItemContentSwipeActionsView {
 
+    public struct Style: Equatable {
+        public enum Shape: Equatable {
+            case rectangle(cornerRadius: CGFloat)
+        }
+
+        public static let standard = Style()
```

Comment:
I'd suggest `.default` - you can use backticks to use it even though it's a keyword
```suggestion
        public static let `default` = Style()
```